lib/test/adapter.js

```diff
@@ -41,14 +41,8 @@ DummyPrompt.prototype.run = function (cb) {
 };
 
 function TestAdapter(answers) {
```

Comment:
Why did you changed that file?

These change are irrelevant to the current matter, and setup method that mutate the object state are pretty hard to maintain on the long run. Please revert those changes